.github/workflows/acceptance-tests.yaml

```diff
@@ -52,3 +52,19 @@ jobs:
           PREFECT_API_URL: ${{ secrets.ACC_TEST_PREFECT_API_URL }}
           PREFECT_API_KEY: ${{ secrets.ACC_TEST_PREFECT_API_KEY }}
           PREFECT_CLOUD_ACCOUNT_ID: ${{ secrets.ACC_TEST_PREFECT_CLOUD_ACCOUNT_ID }}
+      - name: Run acceptance tests (user resource)
```

Comment:
This won't actually run until merged to `main`, so I'll keep an eye out for any failures after this is merged.

Reply:
I did test it locally by passing in the values myself and it passed.